### DISCUSSION TITLE
Is there a way to sumarize vars declared for a task ?

### DISCUSSION THREAD

**Discussion by robinbryce** [go-task/task#617] [Q&A] [answered]
task taskname --sumarize shows me a good view of the task definition. It would help me understand what can or should be parameterized if that view also listed the 'vars' that are declared for the task (+ those at global scope). Is this possible ?

For context, we use task both as a build tool and as a way to provide ergonomics to some infrastructure. We are currently using bot task and go-tusk. The latter for cases where we can't do without 'help' for the options. If we could display the 'vars' for a task - and idealy there default - we think we would probably just use task exclusively.

**Comment by andreynering** [comment, marked as answer]
Hi @robinbryce,

I think that a new issue with a feature request detailing how you see it working with be interesting.

There's currently no builtin command to show do that.
